src/app/components/authorized/post-details/post-details.component.html

```diff
@@ -26,8 +26,15 @@ <h2>{{post.title}}</h2>
         </a>
         </div>
     </div>
+    <div class="createComment">
+        <textarea 
+            maxlength="255"
+            placeholder="this is the place for your non trolling comment..."
+            [(ngModel)]="commentDescription"></textarea>
+        <button (click)="addComment()">Comment</button>
+    </div>
     <div class="commentsWrapper" *ngIf="post.comments.length > 0; else noComments">
-        <div class="comment" *ngFor="let comment of post.comments">
+        <div class="comment" *ngFor="let comment of post.comments.reverse()">
```

Comment:
I've already commented on this in the other PR, reverse() modifies the array. It'd be best to change this.